File: javascript/FizzBuzz.js
const fizzbuzz = (n) => {
    let str;
    for (let i = 1; i < n+1; i++) {
        str = null;
        if (i % 3 === 0)
            str = (str || "") + "Fizz";
        if (i % 5 === 0)
            str = (str || "") + "Buzz";

        if (str)
            console.log(str);
        else
            console.log(i);
    }
}

File: fizz-buzz.py
# fizzbuzz time!
# kwame taylor

def run_fizzbuzz(i):
    '''
    This function runs Kwame's fizzbuzz, which prints out numbers from 1 to i,
    except for each third number it fizz,
    every fifth number it prints buzz,
    every fifteenth number it prints fizzbuzz,
    and every other number prints as itself.
    '''
    fizzbuzz = 'fizzbuzz'
    fizz = 'fizz'
    buzz = 'buzz'

    for i in range(1, i+1):
        if i % 3 == 0 and i % 5 == 0:
            print(fizzbuzz)
        elif i % 3 == 0:
            print(fizz)
        elif i % 5 == 0:
            print(buzz)
        else:

            print(i)

i = 100
run_fizzbuzz(i)